sdk/search/Azure.ResourceManager.Search/api/Azure.ResourceManager.Search.netstandard2.0.cs

```diff
@@ -301,6 +359,7 @@ public enum SearchServiceStatus
         Degraded = 3,
         Disabled = 4,
         Error = 5,
+        Stopped = 6,
```

Comment:
Well this is technically a breaking change - adding a new value to enum type

Reply:
I don't understand, I can remove this if it's a problem

Reply:
Removed using autorest directive